File: src/targets/javascript/fetch/fixtures/nested.js
const url = 'http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value';
const options = {method: 'GET'};

try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/java/nethttp/fixtures/nested.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value"))
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());